app/models/drive_backup.rb

```diff
@@ -55,13 +57,36 @@ def create_folder
 
   def save_backup(filename, path)
     file = drive_service.create_file({ name: filename, parents: [folder.id] },
-                                     fields: "id",
+                                     fields: "id,name",
                                      upload_source: path,
                                      content_type: "application/octet-stream")
-    Rails.logger.info "Backed up database to Drive file #{file.id.inspect}"
+    Rails.logger.info "Backed up database to Drive file #{file.id.inspect}, #{file.name.inspect}"
+  end
+
+  def delete_backups_if_needed
```

Comment:
Do we want to delete all the old backups? maybe save a certain amount in case we need to roll furtherback?